scripts/Inelastic/vesuvio/commands.py

```diff
@@ -570,10 +570,15 @@ def _create_profile_strs_and_mass_list(profile_flags):
     material_builder = MaterialBuilder()
     mass_values, profiles = [], []
     all_mass_values, all_profiles = [], []
-
     for idx, mass_prop in enumerate(profile_flags):
         function_props = ["{0}={1}".format(key, value) for key, value in mass_prop.items()]
-        function_props = ','.join(function_props)
+        # Ensure function_props starts with "function=...,"
+        function_index = [index for index, val in enumerate(function_props) if 'function' in val]
```

Comment:
Is it possible to get 2 `'function='` strings here? The keys in the dictionary should ensure that there is only 1, I think?

In that case we could pop the `function` key out of the `mass_prop` items first then add the rest of the key/value pairs?